docs/development.md

```diff
@@ -310,6 +310,9 @@ docker tag <quarks-job-tag> cfcontainerization/quarks-job:$QUARKS_JOB_IMAGE_TAG
 
 * In PR reviews, make sure only reviewed branches of dependencies are used.
 * If several PRs across repos belong together, be careful to merge them together and in order, so that the master branches stay compatible if one of the merges fails.
+* Make sure that the cf-operator repo PR has updated value of quarks image tag at these places
+  * [Dependency File](https://github.com/cloudfoundry-incubator/cf-operator/blob/master/bin/include/dependencies#L3)
+  * [Helm Chart Value](https://github.com/cloudfoundry-incubator/quarks-job/blob/master/deploy/helm/quarks-job/values.yaml#L5)
```

Comment:
This should not be necessary.

However I think you found a bug.

Here https://github.com/cloudfoundry-incubator/cf-operator/blob/master/bin/build-helm#L19 we should be using `QUARKS_JOB_HELM_VERSION` instead of `ARTIFACT_VERSION`.
However, we don't have a helm repo, so nobody is using that value...

Reply:
Done.